### ISSUE TITLE
Upwards envelopes make muting a subpattern impossible

### ISSUE THREAD

**Issue by coffeevalenbat** [SuperDisk/UGE#118]
Due to the way Cxy is currently handled, once an envelope has been set by an instrument/the main pattern, it's direction can't be changed back from 1 (up) to 0 (down) until a new instrument trigger happens, in ISSO's WIP manual, the reccomended way to kill an instrument's subpattern is using C00, but this coupled with a previous envelope with an upwards direction only makes the envelope reset to volume 0 and crawl up to volume F quickly, a workaround could be using Exx from the main pattern but that'd coggle up the effect row.
![image](https://user-images.githubusercontent.com/34352018/216802650-7506ac45-8948-46d9-9f66-3274f095e50b.png)

**Comment by SuperDisk** [comment]
Yup, the way Cxy is handled right now is bugged and will be fixed.
